### PR TITLE
Bump fluentbit version, change to ubi7/ubi-minimal base image

### PR DESCRIPTION
**Pull request by jewzaam**
### Which issue this PR addresses:

Ev2 cannot build the fluentbit image with base image rhel7

### What this PR does / why we need it:

Switch our fluentbit image from rhel7 base to ubi7/ubi-minimal base.
Bump fluentbit version to 1.7.8-1

### Test plan for issue:

Manually verified on development cluster.

### Is there any documentation that needs to be updated for this PR?

Yes, we need documentation on how we manage fluentbit images.  Current process is a manual release.  That means once this is approved the new image needs to be pushed, ideally before merged, else cluster logging stacks will be broken.  I have added "hold" label.  If merged early it would break only INT, so blast radius is small.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,8 +48,8 @@ image-aro-multistage:
 	docker build --no-cache -f Dockerfile.aro-multistage -t $(ARO_IMAGE) .
 
 image-fluentbit:
-	docker build --no-cache --build-arg VERSION=1.6.10-1 \
-	  -f Dockerfile.fluentbit -t ${RP_IMAGE_ACR}.azurecr.io/fluentbit:1.6.10-1 .
+	docker build --no-cache --build-arg VERSION=1.7.8-1 \
+	  -f Dockerfile.fluentbit -t ${RP_IMAGE_ACR}.azurecr.io/fluentbit:1.7.8-1 .
```
Comment (bryanro92) [inline Makefile:52]: Do we need to change the publish-image-fluentbit version down below? https://github.com/Azure/ARO-RP/pull/1551/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L73

Comment (jewzaam) [reply Makefile]: Yup, thanks.  Moved to a var so we can set it in one place.

**Review by bryanro92 — APPROVED** [review]

**Comment by jewzaam** [comment]
Pushed this version of fluentbit image to INT ACR for E2E to pick up.

/azp run E2E

**Comment by jewzaam** [comment]
Onebranch isn't able to build this.  After current E2E run is complete I will change the base image to ubi7/ubi-minimal to align w/ what is done in Dockerfile.aro-multistage which we know works.  If that still fails we'll dig deeper.

**Comment by jewzaam** [comment]
New version of the 1.7.8-1 image is pushed for E2E to pickup.

**Comment by jewzaam** [comment]
Build works with ubi-minimal on Onebranch, we're good to merge when E2E completes and this is approved.

**Comment by jewzaam** [comment]
Flake on redeploy, may need to look at bumping timeout more.  Portal activity logs indicate this took 4 min 34 seconds from when accepted, but our timeout is 10 min.  So it took longer for it to be available via API for our E2E client.

**Review by bryanro92 — APPROVED** [review]
lgtm

**Comment by jewzaam** [comment]
Holding still to get to public cloud production ACR first.

**Comment by jewzaam** [comment]
Removed hold, image is pushed to production.  Ready to merge.

**Pull request merged**